README.md

```diff
@@ -1 +1,412 @@
+```
+When writing any initializer that is not the default initializer `init`, it is common practice to continue the method name with the word `With` followed by an argument for every property into which the initializer is expected to pass a value. As convention, the arguments in the method declaration are typically named after the property to which it is intended to be assigned.
+
+So, assuming a custom class `FISWarship`:
+
+```obj
+// FISWarship.h
+
+#import <Foundation/Foundation.h>
+
+@interface FISWarship : NSObject
+
+@property (strong, nonatomic) NSString *shipName;
+@property (nonatomic) NSUInteger currentSpeedInKnots;
+@property (nonatomic) NSUInteger maximumSpeedInKnots;
+
+@end
+```
+
+The default initializer for a `FISWarship` class might be declared as:
+
+```objc
+- (instancetype)initWithShipName:(NSString *)shipName
+             currentSpeedInKnots:(NSUInteger)currentSpeedInKnots
+             maximumSpeedInKnots:(NSUInteger)maximumSpeedInKnots;
+```
+
+### Defining the Designated Initializer
+
+#### Autocomplete in Xcode
+
+Because designated initializers, in particular, can produce some ridiculously long method names in Objective-C, relying on autocomplete to reference the declared method elsewhere your in code is good practice to both save keystrokes and to defend against typos.
+
+In the `.m` implementation file for the class, utilize autocomplete by typing the instance method indicator (`-`) followed by the first few letters of the method name. In the case of non-default initializers this will be `initWith...`. Select the appropriate method if there is more than one option and open the curly braces `{` `}` that contain the implementation code.
+
+```objc
+// FISWarship.m
+
+#import "FISWarship.h"
+
+@implementation FISWarship
+
+- (instancetype)initWithShipName:(NSString *)shipName 
+             currentSpeedInKnots:(NSUInteger)currentSpeedInKnots
+             maximumSpeedInKnots:(NSUInteger)maximumSpeedInKnots {
+
+}
+
+@end
+```
+
+#### Implementation Syntax
+
+Now within the implementation of the designated initializer, we want to follow this code structure:
+
+```objc
+{
+    self = [super init];
+    if (self) {
+        /**
+        * Set instance variables to their 
+        * associated method arguments.
+        */
+    }
+    return self;
+}
+```
+This first line `self = [super init]` captures into the `self` keyword a default initialization of an instance of the current class's "super class" (or "parent class")—the class from which the current class inherits. Inheritance will be discussed in a later topic. For now, understand that **only a designated initializer should call the superclass.**
+
+The `if (self) {...}` statement is a convention that came about in order to protect against crashes from improper initializations of the super class. Thanks to ARC (Automatic Reference Counting), this is now an uncommon occurrence, but the practice is widely regarded as a necessary convention and should be included.
```

Comment:
Hm... the `if(self)` stuff and ARC aren't really related. It's guarding against the case that the other initializer you defer to returns nil, which some classes do in the case of bad input to the init method (e.g., `-[NSURL initWithString:]` returns nil if the string is an invalid URL).